src/app/routes.py

```diff
@@ -45,12 +45,20 @@ def fluff_results():
 
     dialect = request.args["dialect"]
     linted = lint(sql, dialect=dialect)
+    fixed_sql = ""
+
+    # Fix can fail badly if linting fails so wrap in a try/except.
+    # See: https://github.com/sqlfluff/sqlfluff-online/issues/25
+    try:
+        fixed_sql = fix(sql, dialect=dialect)
+    except:
```

Comment:
Nit: do we know anything more about the kinds of exceptions that might be raised? Minimally an `Exception`?

Reply:
Any Exception here is a problem. Added `Exception`.

Do wonder why `fix` raises this error as an exception but `lint` just as an unparseable error? But that's more a problem for sqlfluff, than sqlfluff-online and maybe there's a good reason.